### PR TITLE
Add Matus Goljer as author

### PR DESCRIPTION
**Pull request by DamienCassou**

### CODE REVIEW

File: beginend.el [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ;; Copyright (C) 2015-2017 Damien Cassou
 
-;; Author: Damien Cassou <damien@cassou.me>
+;; Authors: Damien Cassou <damien@cassou.me> and Matus Goljer <matus.goljer@gmail.com>
```
Comment (Fuco1) [inline beginend.el]: If there are multiple authors they should be listed on separate lines, see `Author` https://www.gnu.org/software/emacs/manual/html_node/elisp/Library-Headers.html

**Review by Fuco1 — CHANGES_REQUESTED** [review]

File: beginend.el (hunk 1 above)
Comment (Fuco1) [reply beginend.el]: This is "necessary" so that the package tools can properly read the metadata.

Comment (DamienCassou) [reply beginend.el]: Fixed. Thanks

**Comment by DamienCassou** [comment]
Up

**Review by Fuco1 — APPROVED** [review]

**Pull request merged**